src/main/java/com/github/jcustenborder/kafka/connect/spooldir/AbstractCleanUpPolicy.java

```diff
@@ -110,6 +130,25 @@ public void success() throws IOException {
     }
   }
 
+  static class MoveByDate extends AbstractCleanUpPolicy {
+    protected MoveByDate(File inputFile, File errorPath, File finishedPath) {
+      super(inputFile, errorPath, finishedPath);
+    }
+
+    @Override
+    public void success() throws IOException {
+      // Setup directory named as the file created date
+      Path subDirectory = Paths.get(this.finishedPath.getAbsolutePath(), dateFormatter.format(this.inputFile.lastModified()));
```

Comment:
Shouldn't this move files to a subdirectory based on the current timestamp (of when processing is happening) instead of the date on the file?